### ISSUE TITLE
./bootstrap.py fails

### ISSUE THREAD

**Issue by jepler** [ghedo/pflask#10]
```
jepler@rat:~/src/pflask$ ./bootstrap.py 
Downloading http://ftp.waf.io/pub/release/waf-1.8.6...
Traceback (most recent call last):
  File "./bootstrap.py", line 24, in <module>
    waf = urlopen(WAFURL).read()
  File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 469, in error
    result = self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 656, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/lib/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
urllib2.HTTPError: HTTP Error 404: Not Found
```

Host machine is debian jessie amd64

**Comment by jepler** [comment]
This seems to fix it:
-WAFURL     = "http://ftp.waf.io/pub/release/" + WAFRELEASE
+WAFURL     = "http://waf.io/pub/release/" + WAFRELEASE

**Comment by ghedo** [comment]
Thanks! Would you like to open a pull request?

**Comment by jepler** [comment]
This will be resolved when #11 is merged
